modules/tinymce/src/themes/silver/main/ts/ui/dialog/WindowManager.ts

```diff
@@ -190,7 +192,7 @@ const setup = (extras: WindowManagerSetup): WindowManagerImpl => {
           classes: [ ]
         },
         // Fires the default dismiss event.
-        fireDismissalEventInstead: { },
+        fireDismissalEventInstead: (windowParams.persistent ? { event: 'doNotDismissYet' } : { }),
```

Comment:
To make sure I understand correctly, by adding an event here, the code in [Dismissal.ts](https://github.com/tinymce/tinymce/blob/ecdedac778fb742b093edc370e0ee3d2a55fcba9/modules/alloy/src/main/ts/ephox/alloy/sandbox/Dismissal.ts#L53) triggers the event rather than just closing the component?

Reply:
Yeah, passing an empty object would default to `dismissRequested` https://github.com/tinymce/tinymce/blob/aca5f507036544d1af474ec106a4769b9cf2a4b5/modules/alloy/src/main/ts/ephox/alloy/api/ui/InlineView.ts#L230-L232

And when receiving a `dismissRequested`  event, it would `emit` a `formCancelEvent`, closing the dialog.
https://github.com/tinymce/tinymce/blob/aca5f507036544d1af474ec106a4769b9cf2a4b5/modules/tinymce/src/themes/silver/main/ts/ui/dialog/WindowManager.ts#L294-L296

So here we pass an event `doNotDismiss` to prevent the `dismissRequested` from being fired.